x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -177,23 +177,28 @@ const IntegrationPostureDashboard = ({
 export const ComplianceDashboard = () => {
   const [selectedTab, setSelectedTab] = useState(CSPM_POLICY_TEMPLATE);
   const getSetupStatus = useCspSetupStatusApi();
-  const hasFindings = getSetupStatus.data?.status === 'indexed';
+  const hasFindingsKspm =
+    getSetupStatus.data?.kspm?.status === 'indexed' ||
+    getSetupStatus.data?.indicesDetails[0].status === 'not-empty';
+  const hasFindingsCspm =
+    getSetupStatus.data?.cspm?.status === 'indexed' ||
+    getSetupStatus.data?.indicesDetails[0].status === 'not-empty';
   const cspmIntegrationLink = useCspIntegrationLink(CSPM_POLICY_TEMPLATE);
   const kspmIntegrationLink = useCspIntegrationLink(KSPM_POLICY_TEMPLATE);
 
   const getCspmDashboardData = useCspmStatsApi({
-    enabled: hasFindings,
+    enabled: hasFindingsCspm,
   });
   const getKspmDashboardData = useKspmStatsApi({
-    enabled: hasFindings,
+    enabled: hasFindingsKspm,
   });
```

Comment:
Future improvement: consolidate both API calls in a single hook (since the response of both impacts the rendering, that will prevent flickering)